### DISCUSSION TITLE
Welcome to datacook Discussions!

### DISCUSSION THREAD

**Discussion by yorkie** [imgcook/datacook#21] [General]
## 👋 Welcome!

  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by steveoni** [comment]
Hi everyone, my name is Stephen Oni, you can call me steveoni. I'm a machine learning engineer, but currently working as a javascript software engineer @datopian.

I'm currently implementing programs to manage and handle different types of data in datacook.
